### PR TITLE
fetch: create genre calls from API

### PR DESCRIPTION
**Pull request by gutsserrano**
In this branch as created all methods to get the genres and save them in the GenreDAO

### CODE REVIEW

File: src/main/java/com/apollo/model/service/GenreService.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        this.apiService = new ApiService();
+        GenreDao.getInstance();
+    }
+
+    public void getGenres() {
+        ResponseGenreDto retorno = null;
+        var response = apiService.get("/recommendations/available-genre-seeds");
+        ObjectMapper objectMapper = new ObjectMapper();
+        try {
+            retorno = objectMapper.readValue(response, ResponseGenreDto.class);
```
Comment (LucasAlt40) [inline src/main/java/com/apollo/model/service/GenreService.java:26]: Variable in portuguese

File: src/main/java/com/apollo/model/service/GenreService.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+            // Remove all items in the GenreDAO and put the new ones.
+            Collection<Genre> remove = GenreDao.getInstance().getAll();
+            GenreDao.getInstance().getAll().removeAll(remove);
+
+            for (String g : retorno.getGenres()) {
+                GenreDao.getInstance().insert(new Genre(g, 0));
+            }
+
+            // show saved Genres in DAO
```
Comment (LucasAlt40) [inline src/main/java/com/apollo/model/service/GenreService.java:41]: It's best to remove these comments as they don't add anything

File: src/main/java/com/apollo/model/service/GenreService.java [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+        }
+    }
+
+    public void getRecommendations(@PathVariable String genre) {
+        var response = apiService.get("/recommendations?seed_genres=" + genre + "&min_popularity=75");
+
+        //***show the response***
+//        try {
+//            System.out.println(response);
+//        }  catch (Exception e) {
```
Comment (LucasAlt40) [inline src/main/java/com/apollo/model/service/GenreService.java:54]: Same thing here.

**Review by LucasAlt40 — CHANGES_REQUESTED** [review]
It seems good, just remove the comments and it's done.

File: src/main/java/com/apollo/model/service/GenreService.java (hunk 2 above)
Comment (gutsserrano) [reply src/main/java/com/apollo/model/service/GenreService.java]: Ok.

File: src/main/java/com/apollo/model/service/GenreService.java (hunk 3 above)
Comment (gutsserrano) [reply src/main/java/com/apollo/model/service/GenreService.java]: Ok.

File: src/main/java/com/apollo/model/service/GenreService.java (hunk 1 above)
Comment (gutsserrano) [reply src/main/java/com/apollo/model/service/GenreService.java]: I will change to ResponseGenreDto. Thanks!

**Review by LucasAlt40 — APPROVED** [review]

**Pull request merged**